HousingRepairsOnlineApi/Domain/Repair.cs

```diff
@@ -1,10 +1,11 @@
-﻿using Newtonsoft.Json;
+﻿using Amazon.DynamoDBv2.DataModel;
 
 namespace HousingRepairsOnlineApi.Domain
 {
+    [DynamoDBTable("HousingRepairsOnline", LowerCamelCaseProperties = true)]
     public class Repair
     {
-        [JsonProperty("id")]
+        [DynamoDBHashKey]
```

Comment:
As we are maintaining code level compatibility for Azure and AWS, please restore the previous attribute as well as adding the new one too.
```suggestion
        [DynamoDBHashKey]
        [JsonProperty("id")]
```

The deleted using would also need to be restored.